modules/apps/site/site-test/src/testIntegration/java/com/liferay/site/configuration/manager/test/MenuAccessConfigurationManagerPerformanceTest.java

```diff
@@ -43,7 +49,22 @@ public class MenuAccessConfigurationManagerPerformanceTest {
 
 	@Before
 	public void setUp() throws Exception {
-		for (int i = 0; i < _NUMBER_GROUPS; i++) {
+		Class<?> clazz = MenuAccessConfigurationManagerPerformanceTest.class;
```

Comment:
@slnn Can we make it consistent to read the properties and define the fields like `_properties` and `_groupsCount` in `setUpClass` in all the test cases you modify?

Reply:
@slnn If the test class has only 1 `@Test` method then I can accept putting this in `setUp()` and we can move it to `setUpClass` later if we add more `@Test` methods. Please force push if any of the PR needs update.